modules/photons_control/device_finder.py

```diff
@@ -366,10 +367,22 @@ def matches_all(self):
     def points(self):
         """Provide InfoPoints enums that match the keys on this filter with values"""
         for e in InfoPoints:
+            if e.value.condition is not None and not e.value.condition(self):
+                continue
+
             for key in e.value.keys:
                 if self[key] != sb.NotSpecified:
                     yield e
 
+    @property
+    def product_type(self):
+        """Set the product type based on which fields are being filtered"""
```

Comment:
we should base this off the cap on the device rather than what fields have values because the cap is the source of truth and the fields are more a heuristic in the context of a bug making them have values when they shouldn't

Reply:
This is for the Filter() class, and I figured folks would want to filter on device settings, rather than capabilties.

Reply:
the point of the product system is that it's accurate and all encompassing so this should be based off that rather than how it's implemented